build.gradle

```diff
@@ -239,6 +235,36 @@ dependencies {
   compile group: 'org.springframework.boot', name: 'spring-boot-starter-aop'
   compile group: 'org.springframework.boot', name: 'spring-boot-starter-json'
   compile group: 'uk.gov.hmcts.reform', name: 'health-spring-boot-starter', version: '0.0.5'
+//  compile group: 'org.springframework.boot', name: 'spring-boot-starter-data-jpa', version: versions.springBoot
+  compile group: 'org.springframework.boot', name: 'spring-boot-starter-json', version: versions.springBoot
+  compile group: 'org.springframework.boot', name: 'spring-boot-starter-security', version: versions.springBoot
+  //compile group: 'org.springframework.boot', name: 'spring-boot-starter-jdbc'
+  compile group: 'org.springframework.boot', name: 'spring-boot-starter-web', version: versions.springBoot
+  compile group: 'org.springframework.cloud', name: 'spring-cloud-starter-netflix-hystrix', version: versions.springHystrix
+  compile group: 'org.springframework.retry', name: 'spring-retry', version: '1.2.4.RELEASE'
+  compile group: 'org.springframework.boot', name: 'spring-boot-starter-cache', version: versions.springBoot
+  compile group: 'org.springframework.cloud', name: 'spring-cloud-openfeign-core', version: '2.0.0.RELEASE'
+
+
+
+  compile group: 'uk.gov.hmcts.reform', name: 'logging', version: versions.reformLogging
```

Comment:
Same applies to the internal HMCTS dependencies, if it is not part of the initial template we can remove that and add whenever required.